svc/pkg/cluster/worker/src/workers/server_install/install_scripts/files/rivet_fetch_tls.sh

```diff
@@ -0,0 +1,67 @@
+# Write script
+cat << 'EOF' > /usr/bin/rivet_fetch_tls.sh
```

Comment:
would this be cleaner to upload as a separate file in the `files` dir instead of inlining this?

Reply:
We write a script file in `nomad_configure.sh`, i just followed the same format